cg/store/models.py

```diff
@@ -971,3 +972,46 @@ class Order(Base):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class RunDevice(Base):
+    """Model for storing run devices."""
+
+    __tablename__ = "run_device"
+
+    id: Mapped[PrimaryKeyInt]
+    type: DeviceType = mapped_column(types.Enum(*(device.value for device in DeviceType)))
+    device_id: Mapped[int] = mapped_column(ForeignKey("device.id"))
+
+    __mapper_args__ = {
+        "polymorphic_identity": "run_device",
```

Comment:
I think we can skip the `"polymorphic_identity": "run_device"` part on all these. If I've understood it correctly it does the following:

if the type column for entry 123 is equal to "run_device" return an object of type `RunDevice`. Since we never plan to instantiate these models (only inherit from them), we can skip this line.